src/api/container_routes.ts

```diff
@@ -163,7 +164,46 @@ export default class ContainerRoutes {
         })
 
         busboy.on('finish', () => {
-            containerImport.ImportOntology(user, input as ContainerImportT, fileBuffer, req.query.dryrun === 'true')
+            containerImport.ImportOntology(user, input as ContainerImportT, fileBuffer, req.query.dryrun === 'true', false, '')
+            .then((result) => {
+                if (result.isError && result.error) {
+                    res.status(result.error.errorCode).json(result);
+                    return
+                }
+
+                res.status(201).json(result)
+            })
+            .catch((err) => res.status(500).send(err))
+            .finally(() => next())
+        })
+
+        return req.pipe(busboy)
+    }
+
+    private static importUpdatedContainer(req: Request, res: Response, next: NextFunction) {
+        const streamChunks: Buffer[] = []
+        let fileBuffer: Buffer = Buffer.alloc(0)
+        const input: {[key: string]: any} = {}
+        const busboy = new Busboy({headers: req.headers})
+        const user = req.user as UserT
+
+        // if a file has been provided, create a buffer from it
+        busboy.on('file', async (fieldname: string, file: NodeJS.ReadableStream, filename: string, encoding: string, mimeType: string) => {
```

Comment:
should we throw in some protections against the wrong file type?

Reply:
Resolved, let me know if the return error looks appropriate.